### PR TITLE
our works design

### PR DESCRIPTION
**Pull request by v-nakon**
add design

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -284,10 +284,10 @@
   margin-left: 90px;
 }
 
-.services-logo {
+.services-logo, .our-works-logo {
   width: 160px;
   font-family: 'Fira Sans', sans-serif;
-  font-weight: 500;
+  font-weight: 1000;
```
Comment (byzyk) [inline src/style.css]: `1000` is invalid value

File: src/style.css [hunk 2]
```diff
@@ -416,4 +416,59 @@
...
+
+.our-works-text-button {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 52px;
+}
+
+.our-works-text {
+  width: 730px;
+  height: 58px;
```
Comment (byzyk) [inline src/style.css]: `height` is not needed for tet blocks in most cases, like this one

File: src/style.css (hunk 1 above)
Comment (v-nakon) [reply src/style.css]: use `bold`?

Comment (byzyk) [reply src/style.css]: Depending on what you need, `bold` or `700` is the same, the max value of `font-weight` is `900` but not all fonts support it

**Pull request merged**